projects/fsastorefrontlib/src/cms-components/my-account/order/order-details/order-detail-items/order-detail-items.component.spec.ts

```diff
@@ -0,0 +1,127 @@
+import { waitForAsync, ComponentFixture, TestBed } from '@angular/core/testing';
+import { I18nTestingModule, Order } from '@spartacus/core';
+import { OrderDetailsService } from '@spartacus/storefront';
+import { of } from 'rxjs';
+import { FSOrderDetailItemsComponent } from './order-detail-items.component';
+
+const mockProduct = { product: { code: 'test' } };
+
+const mockOrder: Order = {
+  code: '1',
+  statusDisplay: 'Shipped',
+  deliveryAddress: {
+    firstName: 'John',
+    lastName: 'Smith',
+    line1: 'Buckingham Street 5',
+    line2: '1A',
+    phone: '(+11) 111 111 111',
+    postalCode: 'MA8902',
+    town: 'London',
+    country: {
+      isocode: 'UK',
+    },
+  },
+  deliveryMode: {
+    name: 'Standard order-detail-shipping',
+    description: '3-5 days',
+  },
```

Comment:
Remove statusDisplay, deliveryAddress and deliveryMode. We removed these information from the template and also we don't set this type of information in FSA.